### PR TITLE
 Add new FAQ fragment

### LINKED ISSUE #17: Create FAQ fragment
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
No FAQ fragment.

**What you expected to happen**:
Support for an FAQ fragment.

**How to reproduce it (as minimally and precisely as possible)**:
Look for an FAQ fragment.

**Anything else we need to know?**:
Ideas: https://themes.gohugo.io/theme/hugo-universal-theme/faq/

**Environment**:
- Syna Theme version:  v0.7.0
- Hugo version:  v0.30.2

### PR DESCRIPTION
**Pull request by mehrnooshdsht**
**What this PR does / why we need it**:
Add new FAQ fragment and example

**Which issue this PR fixes**:
fixes #17

**Release note**:
```release-note
- faq: Add new fragment called faq to list questions and answers
```

### CODE REVIEW

**Comment by stp-ip** [comment]
Additional note for a follow-on PR:
Currently no faq is indexed by search. We should enable indexing and when clicked the specific faq item should be opened on the page.

File: layouts/partials/fragments/faq.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+      </div>
+      {{- end }}
+      <div class="row text-center justify-content-center">
+        <div class="faq">
+          {{- range .Params.questions }}
+            <div class="card">
+              <div class="card-header" id="{{ .cardId }}">
+                <h5 class="mb-0">
+                  <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#{{ .dataTarget }}"
```
[the remark this answers is not in the document]
Comment (mehrnooshdsht) [reply layouts/partials/fragments/faq.html]: The code is like bootstrap accordion example, But linking the whole header gives a better UX I think.

**Comment by stp-ip** [comment]
I think these would still make for good changes. Always happy to discuss, if you discagree.

* [ ] Fragments page and the example code needs to be updated
* [ ] Example code needs formatting for whitespace. You can take a look at the other code examples.
* [ ] The card additional card border seems unnecessary  `border: 1px solid rgba(0,0,0,.125)`. It leads to a weird small border below the header, if the item isn't open. Happy to discuss this in more detail.
* [ ] linking to a FAQ item should open it right away I think

**Comment by stp-ip** [comment]
Awesome. Thanks for all the fixes. Now it looks great.

We can do the open on linking in another PR.

**Comment by stp-ip** [comment]
Merged \o/

**Comment by mehrnooshdsht** [comment]
So glad I could help :)

**Pull request merged**